### PR TITLE
Add introduction of Alpha feature EvenPodsSpread

### PR DESCRIPTION
**Pull request by Huang-Wei**
Hope it can catch up with the official release timeline.

### CODE REVIEW

**Comment by Huang-Wei** [comment]
/assign @lachie83 

(Sry for this last-minute PR) It'd good to mention this feature in the release note as users have been demanding for this feature for releases.

File: releases/release-1.16/release-notes-draft.md [hunk 1]
```diff
@@ -186,6 +186,7 @@ The main themes of this release are:
...
+- EvenPodsSpread feature has been added in alpha. This feature introduced an API field `pod.spec.topologySpreadConstraints` to enable users control how Pods are spread across the cluster among failure-domains. ([#77327](https://github.com/kubernetes/kubernetes/pull/77327), [#77760](https://github.com/kubernetes/kubernetes/pull/77760), [#77828](https://github.com/kubernetes/kubernetes/pull/77828), [#79062](https://github.com/kubernetes/kubernetes/pull/79062), [#80011](https://github.com/kubernetes/kubernetes/pull/80011), [#81068](https://github.com/kubernetes/kubernetes/pull/81068), [@Huang-Wei](https://github.com/egernst))
```
Comment (idealhack) [inline releases/release-1.16/release-notes-draft.md]: `[@Huang-Wei](https://github.com/Huang-Wei)`

Comment (Huang-Wei) [reply releases/release-1.16/release-notes-draft.md]: oops...

**Comment by saschagrunert** [comment]
Thanks for the clarification @idealhack, I think we also have to update them on kubernetes.io and relnotes.k8s.io. I will take care of that.

File: releases/release-1.16/release-notes-draft.md [hunk 2]
```diff
@@ -186,6 +186,7 @@ The main themes of this release are:
...
+- EvenPodsSpread feature has been added in alpha. This feature introduced an API field `pod.spec.topologySpreadConstraints` to enable users control how Pods are spread across the cluster among failure-domains. ([#77327](https://github.com/kubernetes/kubernetes/pull/77327), [#77760](https://github.com/kubernetes/kubernetes/pull/77760), [#77828](https://github.com/kubernetes/kubernetes/pull/77828), [#79062](https://github.com/kubernetes/kubernetes/pull/79062), [#80011](https://github.com/kubernetes/kubernetes/pull/80011), [#81068](https://github.com/kubernetes/kubernetes/pull/81068), [@Huang-Wei](https://github.com/Huang-Wei))
```
Comment (saschagrunert) [inline releases/release-1.16/release-notes-draft.md]:
```suggestion
- Pod spread constraints have been added in alpha. You can use these constraints to control how Pods are spread across the cluster among failure-domains. ([#77327](https://github.com/kubernetes/kubernetes/pull/77327), [#77760](https://github.com/kubernetes/kubernetes/pull/77760), [#77828](https://github.com/kubernetes/kubernetes/pull/77828), [#79062](https://github.com/kubernetes/kubernetes/pull/79062), [#80011](https://github.com/kubernetes/kubernetes/pull/80011), [#81068](https://github.com/kubernetes/kubernetes/pull/81068), [@Huang-Wei](https://github.com/Huang-Wei))
```

Comment (saschagrunert) [reply releases/release-1.16/release-notes-draft.md]: Related to https://github.com/kubernetes/website/pull/16446/files#r326021937, I've change the note on all relevant PRs, too.

Comment (Huang-Wei) [reply releases/release-1.16/release-notes-draft.md]: @saschagrunert Thanks. I've updated the words to make them consistent.

**Pull request merged**